### DISCUSSION TITLE
The "No Release Judgement" Modifier

### DISCUSSION THREAD

**Discussion by EpiCreeper** [Quaver/Quaver#4807] [Feature Request]
### Is your feature request related to a problem? Please describe.

I personally really like the way LNs are implemented in quaver, but most rhythm games don't require you to release the note in time. Some players transition from other games and find this change challenging and uncomfortable (and others are just bad at the game _cough cough_). These players are likely to turn on NLN, but some maps rely on the long notes to be coherent and interesting, which means NLN players are missing out.

### Describe the solution you'd like

Add a new modifier called No Release Judgement (NRJ) that would change the LN input like so: miss if you released early, marv if you held all the way, even if you didn't release in time. The difficulty of the map would be recalculated as if the NLN modifier is applied.

### Describe alternatives you've considered

So far it seems you dont wanna make any significant changes to the core gameplay, but you could potentially add a new type of LN without release timing, so that mappers could add flavor with LNs without ruining scores. In that case the NRJ mod would just replace al the LNs with the new kind.

### Additional context

I just want people to enjoy LN without being intimidated by the difficulty of them. With this modifier you won't quite get the same experience but I think more people would come to appreciate LN mapping and the effort behind it.

**Comment by ESV-Sweetplum** [comment]
This would be fantastic.
